### PR TITLE
Feature/sqone 643/move post archive settings

### PR DESCRIPTION
**Pull request by nickpelton**
## What does this do/fix?

Moves the Post Archive Settings for Title, Description, and Image under the Post Settings screen. Previously this was a part of "General Settings" which was confusing. Moving it under the Post Menu is a better User Experience.

## QA

Links to relevant issues
- [JIRA TICKET](https://moderntribe.atlassian.net/browse/SQONE-643?atlOrigin=eyJpIjoiZDAwM2EwZTMzZGExNGFlZmE5YTczYjIwYjk0YzY2MjAiLCJwIjoiaiJ9)

Links to deployed, scaffolded demo:
- [Link to Demo](https://sq1-pr804.moderntribe.qa/wp-admin/edit.php?page=edit-php-settings&secret=1)

Screenshots/video:
- [Link to Video](http://p.tri.be/mUAMqq)

## Tests

Does this have tests?

- [x] No, this doesn't need tests because... it's a simple settings screen move.

### CODE REVIEW

File: wp-content/plugins/core/src/Settings/Post_Settings.php [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<?php declare(strict_types=1);
+
+namespace Tribe\Project\Settings;
+
+use Tribe\Libs\ACF\ACF_Settings;
+
+class Post_Settings extends ACF_Settings {
+
+	public function get_title() {
```
Comment (defunctl) [inline wp-content/plugins/core/src/Settings/Post_Settings.php:9]:
```suggestion
	public function get_title(): string {
```

File: wp-content/plugins/core/src/Settings/Post_Settings.php [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+		return __( 'Settings', 'tribe' );
+	}
+
+	public function get_capability() {
```
Comment (defunctl) [inline wp-content/plugins/core/src/Settings/Post_Settings.php:13]:
```suggestion
	public function get_capability(): string {
```

File: wp-content/plugins/core/src/Settings/Post_Settings.php [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+		return 'activate_plugins';
+	}
+
+	public function get_parent_slug() {
```
Comment (defunctl) [inline wp-content/plugins/core/src/Settings/Post_Settings.php:17]:
```suggestion
	public function get_parent_slug(): string {
```

**Review by defunctl — CHANGES_REQUESTED** [review]
Added some suggestions to keep this valid for the v2.0 PHPCS rules

**Comment by nickpelton** [comment]
@defunctl Committed PHPCS changes.

**Review by defunctl — APPROVED** [review]
:rocket:

**Pull request merged**